### ISSUE TITLE
Failed to build Android apk on Ren'Py

### ISSUE THREAD

**Issue by ahr2000** [renpy/renpy#4451]
Hello when I try to compile in Ren'Py for Android I always get an error with this message. 
----------
Ren'Py 8.0.3.22090809

Updating project.

Creating assets directory.

Packaging internal data.

I'm using Gradle to build the package.
To honour the JVM settings for this build a new JVM will be forked. Please consider using the daemon: https://docs.gradle.org/6.1.1/userguide/gradle_daemon.html.
Daemon will be stopped at the end of the build stopping after processing

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'app'.
> Could not resolve all artifacts for configuration ':classpath'.
   > Could not find bundletool-0.13.2.jar (com.android.tools.build:bundletool:0.13.2).
     Searched in the following locations:
         https://dl.google.com/dl/android/maven2/com/android/tools/build/bundletool/0.13.2/bundletool-0.13.2.jar

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 6s

The build seems to have failed.
--------
How can I fix it????

**Comment by renpytom** [comment]
Can you download the file from the location given? It seems most likely that your computer is blocking google's server.

**Comment by ahr2000** [comment]
I have tried to download the file manually by a browser and if I download it, but to do so I have to connect the PC to a VPN but when I try to do it in the Ren'Py I repeat the same error

**Comment by ahr2000** [comment]
I have tried to download the file manually by a browser and if I download it, but to do so I have to connect the PC to a VPN but when I try to do it in the Ren'Py I repeat the same error

**Comment by renpytom** [comment]
Please make sure that Ren'Py (and gradle's) requests go through your VPN. 

You can check this by going to the shift+O console, and typing:

```
import requests
requests.get("https://dl.google.com")
```

**Comment by ahr2000** [comment]
I get this that means, just and I'm novice in this
---
import requests
requests.get("https://dl.google.com")

File "<string>", line 1:expected statement.
   import requests
requests.get("https://dl.google.com")

**Comment by renpytom** [comment]
Did you do it in the console?

**Comment by ahr2000** [comment]
From Ren'Py, yes

**Comment by renpytom** [comment]
Works for me. 

You could also put:

```renpy
init python:
     import requests
     request.get("https://dl.google.com")
```

In a script.

**Comment by ahr2000** [comment]
I tried the first option again and it didn't give me an error (it seems a letter was missing) and put me <Response [200]>, what does that mean

**Comment by danac78** [comment]
TLDR: It meant that you reached a HTTP OK if I remember that HTTP code right.

**Comment by MadServlet** [comment]
thats a valid response. it means it successfully received your http request.
try to unwrap the response and get the content.

**Comment by qinlili23333** [comment]
if ur using windows:
```
set HTTP_PROXY=http://127.0.0.1:10809(Change it to ur vpn http proxy conf)
set HTTPS_PROXY=http://127.0.0.1:10809(Change it to ur vpn http proxy conf)
start renpy.exe
```
save these as a file named as start.bat, put it in renpy folder, run it, and renpy now can use vpn

when I worked in a location that need vpn to access google last year, I used this script to make it work

similar way under linux, just change the cmd style to linux sh style

**Comment by ahr2000** [comment]
I will try all variants

**Comment by renpytom** [comment]
I'm closing this - feel free to re-open it if you're still having problems.
